docs/patterns/branching_discretechannels_[3.12.1,).partial.md

```diff
@@ -0,0 +1,3 @@
+To prevent [retention policies](/docs/administration/retention-policies/index.md) for one channel from impacting deployments for another channel, you should set the `Discrete Channel Release` flag at under `Deployment Target settings` on the {{Project,Process}} page. Enabling this feature will also ensure that your project overview dashboard correctly shows which releases are current for each environment _in each channel_. Without this set, the default behaviour is for releases across channels to supersceed each other, for example in a hotfix scenario where the `3.2.2-bugfix` is expected to override the `3.2.2` release which may then be considered for retention policy cleanup. 
```

Comment:
screenshot says 'discrete channel releases' (missing trailing 's' in the words.
typo in 'supersceed' - 'supersede'

`Without this set, the default behaviour is for releases across channels to supersceed each other, for example in a hotfix scenario where the `3.2.2-bugfix` is expected to override the `3.2.2` release which may then be considered for retention policy cleanup. ` => `Without this set, the default behaviour is for releases across channels to supersede each other (for example, in a hotfix scenario where the `3.2.2-bugfix` is expected to override the `3.2.2` release, allowing `3.2.2` to be considered for retention policy cleanup).`